pystarport/cosmoscli.py

```diff
@@ -211,7 +211,13 @@ def block_time(self):
     def balances(self, addr, height=0):
         return json.loads(
             self.raw(
-                "query", "bank", "balances", addr, height=height, home=self.data_dir
+                "query",
+                "bank",
+                "balances",
+                addr,
+                height=height,
+                output="json",
```

Comment:
why need this?

Reply:
you mean output as json?

Reply:
it's not needed before because it's always in `client.toml`, otherwise we need to specify node_rpc explicitly as well.

Reply:
I see, seems not respect client.toml now

Reply:
can wait a bit https://github.com/cosmos/cosmos-sdk/pull/20033

Reply:
need to port to our fork?